### ISSUE TITLE
Steam Heavy description

### ISSUE THREAD

**Issue by LuxArdens** [LuxArdens/Stellaris---RPG-Traits-mod#37]
@TvoEx 

Saw the steam question for what Heavy really does. You can paste this description there if you want:

The Heavy version is a heavily expanded version of RPG Traits. While the Core version only overhauls Traits and incompatibility is mostly limited to trait-related mods, the Heavy version goes completely loose and overhauls a more things to allow for more creativity in designing species. The main things that have been overhauled are:

- Colonization & Planet classes
- Terraforming
- Planet Modifiers

E.g. in Heavy you can start on an Asteroid, or live on Gas Giants. You can even play as a Star-inhabiting species. In addition there are some fancy new late-game techs which allow you to do crazy stuff like terraform Gas Giants and Molten worlds (those are stupendously costly though). It's specifically made to go nuts with your sci-fi fantasy. The only drawback to this version is lack of compatibility with mods that also modify these aspects. In addition, Heavy rebalances ALL Habitability modifiers, so it matters much more. This means many mods that add habitability modifiers in some way (which are tailored to vanilla scaling with bonuses like +10% being very common) will be very exploit-y if used with Heavy.

In summary: pick Heavy if you want even MOAR freedom when designing your races, or if you feel like the 9 vanilla planet types are boring af.

**Comment by TvoEx** [comment]
It has been done.
